cmd/cmd_util.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright © 2021 The Persistent-Volume-Migrator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"strings"
+
+	"persistent-volume-migrator/internal/ceph/rbd"
+	"persistent-volume-migrator/internal/kubernetes"
+	logger "persistent-volume-migrator/internal/log"
+
+	v1 "k8s.io/api/core/v1"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+var (
+	err error
```

Comment:
This is not really idiomatic, why do we need this? It's better to keep `err` within each function's scope.